cmd/ethrex/bench/import_blocks_benchmark.rs

```diff
@@ -1,21 +1,22 @@
 use criterion::{criterion_group, criterion_main, Criterion};
 use ethrex::{
     cli::{import_blocks, remove_db},
-    utils::set_datadir,
+    utils::{set_data_sub_dir, set_datadir},
     DEFAULT_DATADIR,
 };
 use ethrex_vm::EvmEngine;
 
 #[inline]
 fn block_import() {
     let data_dir = DEFAULT_DATADIR;
-    set_datadir(data_dir);
+
+    let network = "../../test_data/genesis-perf-ci.json";
+
+    set_datadir(&set_data_sub_dir(data_dir, &Some(network.to_string())));
```

Comment:
this is weird imo. it's like every time you call `set_datadir ` you need to call `set_data_sub_dir`.